Articles/WhatIs/WhatIsSoftwareProcessImprovement.md

```diff
@@ -0,0 +1,39 @@
+### What is Software Process Improvement?
+#### Contributed by  [Roscoe A. Bartlett](https://github.com/bartlett)
+#### Publication date: April 28, 2023
+
+<!--- deck start --->
+Software process improvement is the application of a systematic approach to the identification, evaluation, and improvement of software processes which is critical for computational science and engineering projects.
+<!--- deck end --->
+
+<!--- body start --->
+
+The goal of **Software Process Improvement** is a systematic methodology to improve the quality, efficiency, and effectiveness of software development and maintenance.
+Improvement efforts can implemented in a variety of ways, but it typically involve the following steps:
```

Comment:
There are various ways to implement improvement efforts, but they generally involve the following steps:

Reply:
Implemented in 6ac58e02bd4dfc4b6697191196fe87cbd88cad64.